src/helpers/reviews.js

```diff
@@ -1,15 +1,23 @@
 import fetch from "node-fetch";
 
-const API_URL_REVIEWS = "https://plankton-app-xhkom.ondigitalocean.app/api/reviews";
+const API_URL_REVIEWS =
+  "https://plankton-app-xhkom.ondigitalocean.app/api/reviews";
 
-export async function loadMovieReviews(id) {
-  const res = await fetch(`${API_URL_REVIEWS}?filters[movie]=${id}`);
+export async function loadMovieReviews(id, query) {
+  let res = "";
```

Comment:
To me it's confusing to give this variable a string value (`""`). I would probably refactor this function to not need this variable at all, but if I was to keep it, I would declare it as just `let res`, without a value.